echopype/tests/convert/test_convert_azfp.py

```diff
@@ -172,3 +174,38 @@ def test_convert_azfp_01a_notemperature_notilt(azfp_path):
     assert "tilt_y" in echodata["Platform"]
     assert echodata["Platform"]["tilt_x"].isnull().all()
     assert echodata["Platform"]["tilt_y"].isnull().all()
+
+
+def test_load_parse_azfp_xml(azfp_path):
```

Comment:
The test looks great! Could you add more tests for the values, especially those that are across all 4 frequencies? Just so that we have a piece of mind that things are 100% correct. Thanks!

Reply:
Added new tests for other parameters!